.github/workflows/CI.yml

```diff
@@ -131,7 +135,7 @@ jobs:
       # Upload Regression Results as Stable if Scheduled or Push to Main
       - name: Upload Results as Reference
         # upload the results for scheduled CI (on main) and pushes to main
-        if: github.ref == 'refs/heads/main'
+        if: github.ref == 'refs/heads/main' && runner.os =='ubuntu'
```

Comment:
I guess this means only to upload the results from the ubuntu tests since they should be the same on either platform?

Reply:
Yes - we treat the results on Linux as 'correct' and compare MacOS to that. Since we distribute on a Linux image for docker, I have taken that as the absolute "ground truth".

The results should never differ in the first place (hopefully).